package.json

```diff
@@ -54,6 +55,9 @@
 		"sinon": "^9.0.0",
 		"tsd": "^0.11.0",
 		"uuid": "^7.0.2",
-		"xo": "^0.27.2"
+		"xo": "^0.27.2",
```

Comment:
Try upgrading it to the latest version. That version has built-in TS support.

Reply:
The current version already does have that commit. And to quote a common saying, "it works on my machine"...

Could be travis CI caching an old version in the `node_modules` for some other dep though.

Reply:
Sure, but the latest XO version upgrades the TS parsers and such.